### PR TITLE
Update sampleapp 4.23.0

### PR DESCRIPTION
**Pull request by christianbilodeau**

### CODE REVIEW

File: SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -526,7 +526,7 @@
 			repositoryURL = "https://github.com/ContentSquare/CS_iOS_SDK.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 4.22.1;
+				minimumVersion = 4.23.0;
```
Comment (zouhair-isk) [inline SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj:529]: nit: I still don't agree with having the minimum version as the latest version while having `upToNextMajorVersion`

**Review by zouhair-isk — APPROVED** [review]

File: SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj (hunk 1 above)
Comment (christianbilodeau) [reply SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj:529]: Yeah I agree with you, shall we just set it to `Exact` ?

Comment (zouhair-isk) [reply SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj:529]: yes sir, exact would make sense in this case as we change it manually every release but it is a Nit, we can ignore my comment if it doesn't make that much sense here

[some background story](https://contentsquare.slack.com/archives/C02HU09E0JG/p1683880402946659)

Comment (christianbilodeau) [reply SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj]: set to exact

**Review by zouhair-isk — APPROVED** [review]
LGTM, 
Nit: but wouldn't be maintenance free if we have a link to the public doc instead of writing a description?

**Comment by christianbilodeau** [comment]
@zouhair-isk There's already a link to the doc at the bottom of the README, I think it's nice to have a description that's closely tied to the screens the user will see in the sample app (especially since they're not perfect). Maybe later when we improve the sample app as a whole we'll be able to do without the description

**Review by mathieu-ouvrard — APPROVED** [review]

**Pull request merged**